lib/Experian/IDAuth.pm

```diff
@@ -152,19 +155,42 @@ sub _build_request {
     my $self = shift;
 
     $self->{request_as_xml} =
-          '<Search>'
+          '<xml><![CDATA[<Search>'
         . $self->_build_authentication_tag
         . $self->_build_country_code_tag
         . $self->_build_person_tag
         . $self->_build_addresses_tag
         . $self->_build_telephones_tag
         . $self->_build_search_reference_tag
         . $self->_build_search_option_tag
-        . '</Search>';
+        . '</Search>]]></xml>';
 
     return 1;
 }
 
+# This is built based Section 3b on the User Guide we received from Experian avaialble here : https://github.com/regentmarkets/third_party_API_docs/blob/master/AML/User%20Guide%20for%202FA.pdf
```

Comment:
Could you split this comment across two lines, putting the URL on the next line so it's easier readable at 80 columns?